modules/tv/search.php

```diff
@@ -281,7 +281,7 @@
             }
         // If the next thing starts with stars, it's a movie rating query
             if (preg_match('#(\\*+\s*(1/2\b|0?\.5\b|-)?)\s*(.*?)$#', $search_str, $match)) {
-                $starcount = substr_count($match[1], '*') / 4;
+                $starcount = substr_count($match[1], '*') / 5;
```

Comment:
It's counting the number of asterixes entered in the search form. This patch just normalizes the maximum stars number to 5 stars, which is used anywhere else in mythweb.